test/integration/sequelize/query.test.js

```diff
@@ -10,7 +10,7 @@ const moment = require('moment');
 const { DatabaseError, UniqueConstraintError, ForeignKeyConstraintError } = Support.Sequelize;
 
 const qq = str => {
-  if (dialect === 'postgres' || dialect === 'mssql') {
+  if (dialect === 'postgres' || dialect === 'mssql' || dialect === 'db2') {
```

Comment:
```suggestion
  if (['postgres', 'mssql', 'db2'].includes(dialect)) {
```

If you change it in one location, it would be better to change it in other places as well.

Reply:
Done. A similar change needs to be done at line 16. I did not do it as it is not a db2 code.

Reply:
Agree, I will refactor the others in a separate PR later this week

Reply:
I have refactored the others and that has been merged, can you rebase?